src/js/vic-v2/components/IntroductionPage.jsx

```diff
@@ -60,7 +60,7 @@ class IntroductionPage extends React.Component {
               <div className="vic-intro-indent">
                 <div><h6>The photo has to follow all the standards listed here. It must:</h6></div>
                 <ul>
-                  <li>Show a full front view of your face and neck, with full visibility of the face and no shadows, <strong>and</strong></li>
+                  <li>Show a full front view of your face and neck (with no hat, head covering, or headphones covering or casting shadows on your hairline or face, <strong>and</strong></li>
```

Comment:
This is missing a closing paren, I think.

Reply:
Fixed